FrugalApp/server.js

```diff
@@ -0,0 +1,49 @@
+// server.js
```

Comment:
`app.js` is analogous with `server.js` in your case. What you've done here is created a totally separate server for the API to live on.